content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,9 +15,21 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-You have two mutually exclusive options to install Istio:
-- To use Kubernetes manifests to deploy Istio, follow the instructions for [option #1](#option-1).
-- To use [Helm's Tiller pod](https://helm.sh/) to manage your Istio release, follow the instructions for [option #2](#option-2).
+The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
+
+1.  Update Helm's dependencies:
+
+    {{< text bash >}}
+    $ helm repo add istio.io "https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-latest-daily/charts/"
+    $ helm dep update install/kubernetes/helm/istio
+    {{< /text >}}
+
+1. Choose one of the following two **mutually exclusive** options described below.
+
+    {{< tip >}}
+    To customize Istio and install addons, use the `--set <key>=<value>` option in the helm template or install command. [Installation Options](/docs/reference/config/installation-options/) references supported installation key and value pairs.
```

Comment:
This tip would be better placed as an additional paragraph before the Prerequisites section since it points readers to additional information regarding the available options.